templates/vm/ssc600.xml.j2

```diff
@@ -0,0 +1,116 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--  Copyright (C) 2024 Savoir-faire Linux, Inc-->
+<!-- SPDX-License-Identifier: Apache-2.0 -->
+
+{% set cpu_nb = (vm.cpuset | length) if vm.cpuset is defined else vm.nb_cpu | default(1) %}
+
+<domain type="kvm">
+    <name>{{ vm.inventory_hostname }}</name>
```

Comment:
You should add scc600 somewhere in the name or explicitly explain that the VM's name must contain scc600. Otherwise, the hook will ignore.